publication-commons/src/main/java/no/unit/nva/publication/permissions/publication/grant/EditorGrantStrategy.java

```diff
@@ -18,15 +18,24 @@ public boolean allowsAction(PublicationOperation permission) {
         if (!hasAccessRight(MANAGE_RESOURCES_ALL)) {
             return false;
         }
+        if (publicationChannelIsClaimed() && !userBelongsToPublicationChannelOwner()) {
+            return false;
+        }
         return switch (permission) {
             case UPDATE, PARTIAL_UPDATE -> true;
-            case UNPUBLISH -> userRelatesToPublicationThroughPublicationOwnerOrCuratingInstitution() && isPublished();
-            case REPUBLISH, TERMINATE -> userRelatesToPublicationThroughPublicationOwnerOrCuratingInstitution() && isUnpublished();
+            case UNPUBLISH ->
+                userRelatesToPublicationThroughPublicationOwnerOrCuratingInstitutionOrChannelClaim() && isPublished();
```

Comment:
ah man, time to make it shorter maybe? userRelatesToPublication is maybe enough?